docs/cheaha/migration_overview.md

```diff
@@ -0,0 +1,194 @@
+
+In rare cases, tools compiled on on Intel processor can cause `Illegal Instruction` errors when run on AMD CPUs. If your jobs were submitted to one of the mainline partitions and show this error after migration, please [contact support](../index.md#how-to-contact-us). As the migration continues, the standard Intel nodes will also be migrated from GPFS 4 to GPFS 5 and added to the mainline partitions.
+
+#### Compute Migration
+
+To best accommodate workloads on both clusters during data migration, compute nodes will be migrated in two stages: at 50% and 100% data migration. Due to hardware networking and rack constraints, we have limited options as far as which nodes we can move at which times. See below for a list of which nodes will be migrated to GPFS 5 at which time.
+
+**50% Migration Completion**:
+
+- All pascalnodes (18 total nodes on GPFS 5, 72 available P100s)
+- 10 amperenodes (15 total on GPFS 5, 30 available A100s)
+
+**100% Migration Completion**:
+
+- 5 amperenodes (20 total on GPFS 5)
+- All largemem nodes (14 total on GPFS 5)
+
+### Directory Paths
+
+We have reorganized our GPFS 5 filesets to improve file access performance specifically for our network scratch space. As part of this, we have changed the `/data` drive to `/gpfs` and have brought `/scratch` under it.
+
+**Path conversions from GPFS 4 to GPFS 5**:
+
+- `/data/user` -> `/gpfs/user`
+- `/data/user/home` -> `/gpfs/user/home`
+- `/data/project` -> `/gpfs/project`
+- `/scratch` -> `/gpfs/scratch`
+
+In order to smooth the transition to GPFS 5, we have provided symlinks that mimic the GPFS 4 directory structure:
+
+**GPFS 5 symlinks**:
+
+- `/data` -> `/gpfs`
+- `/scratch` -> `/gpfs/scratch`
+- `/home` -> `/gpfs/user/home`
+
+<!-- markdownlint-disable MD046 -->
+!!! critical
+    It is **not** necessary to immediately make changes to your paths in your scripts. The provided symlinks ensure backwards compatibility with GPFS 4 path structure in existing scripts in 99% of cases. Requeued and pending jobs submitted using GPFS 4 paths will not be impacted by these path changes.
+<!-- markdownlint-enable MD046 -->
+
+As you use GPFS 5, we suggest switching to the new path structure in your scripts as the symlinks will eventually be deprecated.
+
+### Quotas
+
+Existing quotas on project spaces have not been altered as part of the migration in order to smoothly accommodate existing data. Quotas will be enforced as the sum of storage used across both GPFS and CephFS. Future changes to quotas will be communicated when appropriate.
+
+### Scratch
+
+Network scratch space (`/scratch` or `/gpfs/scratch`) will have a couple of major changes moving forward
+
+1. Total `/scratch` capacity will be reduced from 800 TiB to **500 TiB** for all users. This is a consequence of a necessary reduction in total GPFS capacity.
```

Comment:
I believe the final number is 448 TiB.

Reply:
```
mkdefender42@gmail.com@login006:~$ df -h /scratch
Filesystem      Size  Used Avail Use% Mounted on
gpfs            500T   39T  462T   8% /scratch
```

I'm fairly sure `df` reports in base-2 units, and it's showing 500 TiB total

Reply:
@jprorama Is the 500 TiB shown in `df` just the NVMe tier of storage? Do we want to just remove the scratch section for now until we have the data removal policy solidified and ready for deploy?

Reply:
I'm going to leave as-is for now since user's can also check `df` if they think about it. I'd rather this information match what the system reports regardless of what specific tiering is implemented on scratch